### PR TITLE
feature/CWALL-219

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+"GuestCredential"
+],
+"expirationDate": "{{{ dateTimeAfterDays 7 }}}",
+"credentialSubject": {
+"firstName": "{{firstName}}",
+"lastName": "{{lastName}}",
+"email": "{{email}}",
+"type": "Sphereon Guest"
+}
+}
```
Comment (zoemaas) [inline packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs]: Trailing new line missing

File: packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts [hunk 2]
```diff
@@ -199,7 +203,7 @@ export interface SSICredentialIssuedSuccessPageConfig extends PageConfig {
     rightPaneButtonHeight?: string
 }
 
-export interface SSICredentialsLandingPageConfig extends PageConfig {
+export interface SSICredentialsLandingPageConfig extends PageConfig { // TODO
```
Comment (zoemaas) [inline packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts:206]: Remove the comment or add a reference to a ticket meant to handle the issue

File: packages/oid4vci-demo-frontend/src/pages/SSIInformationSuccessPage/index.tsx [hunk 3]
```diff
@@ -38,13 +38,37 @@ const SSIInformationSuccessPage: React.FC = () => {
...
+                            ...(pageConfig.photoLeft && {
+                                background: `url(${pageConfig.photoLeft})`,
+                                backgroundSize: 'cover',
+                            }),
+//                            height: '100%',
+                            width: pageConfig.leftPaneWidth ?? 'auto',
+                            height: pageConfig.leftPaneWidth ? '100%' : 'auto',
+                            alignItems: 'center',
+//                            ...((pageConfig.photoLeft || pageConfig.photoLeftManual) && { background: `url(${state?.isManualIdentification? `${pageConfig.photoLeftManual}` : `${pageConfig.photoLeft}`}) 0% 0% / cover`}),
+                            ...(pageConfig.backgroundColor && { backgroundColor: pageConfig.backgroundColor }),
```
Comment (zoemaas) [inline packages/oid4vci-demo-frontend/src/pages/SSIInformationSuccessPage/index.tsx:49]: Remove commented-out code

File: packages/oid4vci-demo-frontend/src/router/actions/credential-actions.ts [hunk 4]
```diff
@@ -33,9 +34,24 @@ export const createCredentialOffer = async (actionParams: Record<string, any>, s
...
+    //             'pre-authorized_code': shortUuid,
+    //             user_pin_required: false,
+    //         },
+    //     },
+    //     credentialDataSupplierInput: {
+    //         ...state.payload
+    //     },
+    //     credentials: [state.credentialType ?? ("issueCredentialType" in actionParams ? actionParams.issueCredentialType : generalConfig.issueCredentialType)],
+    // })
+
```
Comment (zoemaas) [inline packages/oid4vci-demo-frontend/src/router/actions/credential-actions.ts]: Remove commented-out code

File: packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts [hunk 5]
```diff
@@ -51,26 +55,32 @@ export const getCredentialBrandings = async (metadata: EndpointMetadataResult):
...
-            const types = 'types' in credentialsSupported ? credentialsSupported.types : undefined
-            if (types) {
-                const credentialTypes: Array<string> =
-                    types.length > 1
-                        ? types.filter((type: string) => type !== 'VerifiableCredential')
-                        : types.length === 0
-                            ? ['VerifiableCredential']
-                            : types
+                const types = 'types' in credentialsConfigSupported // TODO credentialsConfigSupported.types is deprecated
+                    ? credentialsConfigSupported.types as string[]
```
Comment (zoemaas) [inline packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts:68]: Remove the comment or add a link to the ticket meant to handle it

**Review by zoemaas — DISMISSED** [review]
Left small remarks

File: packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs (hunk 1 above)
Comment (BtencateSphereon) [reply packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs]: added

File: packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts (hunk 2 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts:206]: added ticket, we do not have a project for this so it is CWALL

File: packages/oid4vci-demo-frontend/src/pages/SSIInformationSuccessPage/index.tsx (hunk 3 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/pages/SSIInformationSuccessPage/index.tsx:49]: removed

File: packages/oid4vci-demo-frontend/src/router/actions/credential-actions.ts (hunk 4 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/router/actions/credential-actions.ts]: removed

File: packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts (hunk 5 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts:68]: added ticket, we do not have a project for this so it is CWALL

File: packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts (hunk 2 above)
Comment (zoemaas) [reply packages/oid4vci-demo-frontend/src/ecosystem/ecosystem-config.ts:206]: Please add a link to the ticket in the comment

File: packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts (hunk 5 above)
Comment (maikel-maas) [reply packages/oid4vci-demo-frontend/src/utils/mapper/branding/OIDC4VCIBrandingMapper.ts:68]: Please add a link to the ticket in the comment

File: packages/agent/conf/demos/vismayuki/templates/sphereonGuest/GuestCredential.hbs [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+"GuestCredential"
+],
+"expirationDate": "{{{ dateTimeAfterDays 7 }}}",
+"credentialSubject": {
+"firstName": "{{firstName}}",
+"lastName": "{{lastName}}",
+"email": "{{email}}",
+"type": "Sphereon Guest"
+}
+}
```
Comment (zoemaas) [inline packages/agent/conf/demos/vismayuki/templates/sphereonGuest/GuestCredential.hbs]: Trailing new line missing

File: packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (sksadjad) [inline packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs:1]: fix the indentations here

File: packages/agent/conf/demos/vismayuki/templates/sphereonGuest/GuestCredential.hbs [hunk 8]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (sksadjad) [inline packages/agent/conf/demos/vismayuki/templates/sphereonGuest/GuestCredential.hbs]: fix the indentations here, plus empty line at the end

File: packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json [hunk 9]
```diff
@@ -123,5 +123,13 @@
...
   "web_wallet_address": "Web wallet adres",
-  "go": "Ga"
+  "go": "Ga",
+  "vismayuki_landing_page_title": "Beschikbare verificatie flows",
+  "vismayuki_landing_page_description": "Via deze demo kan er om verschillende credentials gevraagd worden. Kies één optie.",
+  "vismayuki_verify_top_title": "Graag ontvangen we van u 1 of meer credentials",
+  "vismayuki_verify_top_description": "Reden: {{purpose}}",
+  "vismayuki_verify_bottom_description": "Scan deze QR-code met u mobiele wallet’s QR-code scanner. \nOf vul het adres van uw web wallet in om uw web wallet te openen.",
+  "konkuk_share_success_top_title": "Thanks for submitting the information",
+  "konkuk_share_success_top_description": "This has been received successfully.",
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json]: translate to Dutch

File: packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json [hunk 10]
```diff
@@ -123,5 +123,13 @@
...
+  "konkuk_share_success_button_caption": "Return to first pages"
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json]: translate to Dutch

File: packages/oid4vci-demo-frontend/src/pages/SSICredentialVerifyRequestPage/index.tsx [hunk 11]
```diff
@@ -140,7 +188,7 @@ export default function SSICredentialVerifyRequestPage(): React.ReactElement | n
                               <Text style={{flexGrow: 1, color: `${pageConfig.rightPaneLeftPane?.qrCode?.bottomText?.fontColor}`, ...(!!pageConfig.rightPaneLeftPane?.qrCode?.topTitle && { marginTop: '12%' })}}
                                     pStyle={pageConfig.rightPaneLeftPane?.qrCode?.bottomText?.pStyle}
                                     className={`${style.pReduceLineSpace} ${pageConfig.rightPaneLeftPane?.qrCode?.bottomText?.className ?? 'poppins-semi-bold-16'}`}
-                                    title={t(`${pageConfig.rightPaneLeftPane?.qrCode?.bottomText?.credential_verify_request_right_pane_bottom_title}`).split('\n')}
+                                    title={pageConfig.rightPaneLeftPane?.qrCode?.bottomText?.credential_verify_request_right_pane_bottom_title ? t(`${pageConfig.rightPaneLeftPane?.qrCode?.bottomText?.credential_verify_request_right_pane_bottom_title}`).split('\n') : []}
```
Comment (sksadjad) [inline packages/oid4vci-demo-frontend/src/pages/SSICredentialVerifyRequestPage/index.tsx:191]: this seems incorrect. I think it should be translated

**Review by sksadjad — CHANGES_REQUESTED** [review]
added a few (small) notes

File: packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs (hunk 7 above)
Comment (BtencateSphereon) [reply packages/agent/conf/demos/konkuk/templates/sphereonGuest/GuestCredential.hbs:1]: fixed

File: packages/agent/conf/demos/vismayuki/templates/sphereonGuest/GuestCredential.hbs (hunk 8 above)
Comment (BtencateSphereon) [reply packages/agent/conf/demos/vismayuki/templates/sphereonGuest/GuestCredential.hbs]: removed the file as i don't think this demo uses them

File: packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json (hunk 9 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json]: fixed

File: packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json (hunk 10 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json]: fixed

**Review by sksadjad — APPROVED** [review]
LGTM

**Pull request merged**